docs/developer/how_to_contribute.rst

```diff
@@ -211,6 +209,18 @@ Code contribution steps review
 #.  Create a new PR in GitHub
 #.  Respond to any code review feedback
 
+Blueprints
+----------
+
+We have a type of document used to describe our architectural choices, called 
+‘blueprint’. This document is developed for our developers and records all the
+main architectural decisions made in the kytos project. Blueprints are used to
+specify the details of any process, resource, and anything else that is
+desirable to be implemented, resulting in a major architectural change. You can
+check out our blueprints to learn more about our architectural choices in the
+`blueprints section`_. Remember that this document is not just for you, but
+mainly for other developers.
```

Comment:
Please change "developed for our developers", something like "written for" or "the intended audience are the developers and users of Kytos". It's also important to state that blueprints can be written by anyone, as they are "enhancement proposals" which can be accepted or not.